__tests__/components/TransactionList/TransactionRow.spec.ts

```diff
@@ -0,0 +1,888 @@
+        return new MultisigAccountModificationTransaction(
+            NetworkType.TEST_NET,
+            1,
+            mockDeadline,
+            UInt64.fromUint(100),
+            2,
+            1,
+            addressAdditions,
+            [],
+            mockSignature,
+            PublicAccount.createFromPublicKey(currentSigner.publicKey, NetworkType.TEST_NET),
+        );
+    };
+
+    const getTransactionRowWrapper = (state = {}, props = {}) => {
+        const mockNetworkStore = {
+            namespaced: true,
+            state: { networkConfiguration: undefined },
+            getters: {
+                networkConfiguration: () => Object.assign(new NetworkConfigurationModel(), { epochAdjustment: 1573430400 }),
+            },
+        };
+
+        const mockMosaicStore = {
+            namespaced: true,
+            state: {
+                balanceMosaics: [],
+                networkMosaicId: null,
+                holdMosaics: [],
+            },
+            getters: {
+                balanceMosaics: (state) => {
+                    return state.balanceMosaics;
+                },
+                networkMosaic: (state) => {
+                    return new MosaicId(state.networkMosaicId);
+                },
+                holdMosaics: (state) => {
+                    return state.holdMosaics;
+                },
+            },
+        };
+
+        const mockAccountStore = {
+            namespaced: true,
+            state: {
+                currentAccount: null,
+                multisigAccountGraph: null,
+                currentAccountMultisigInfo: null,
+            },
+            getters: {
+                currentAccount: (state) => {
+                    return state.currentAccount;
+                },
+                multisigAccountGraph: (state) => {
+                    return state.multisigAccountGraph;
+                },
+                currentAccountMultisigInfo: (state) => {
+                    return state.currentAccountMultisigInfo;
+                },
+            },
+        };
+
+        const mockProfileStore = {
+            namespaced: true,
+            state: {
+                currentProfile: null,
+            },
+            getters: {
+                currentProfile: (state) => {
+                    return state.currentProfile;
+                },
+            },
+        };
+
+        return getComponent(
+            TransactionRow,
+            {
+                mosaic: mockMosaicStore,
+                network: mockNetworkStore,
+                profile: mockProfileStore,
+                account: mockAccountStore,
+            },
+            state,
+            props,
+            {
+                Tooltip: true,
+            },
+            dispatch,
+        );
+    };
+
+    describe('transaction icon column', () => {
+        // Todo: Currently unable return icon url.
```

Comment:
why not ?

Reply:
I was not able to get the source path, but now I got that path from `wrapper.html()` I will update the test.